server/connection_manager.cpp

```diff
@@ -53,87 +86,199 @@ static bool CommandProcessor(int socket, char * buffer, void * dacs, int ndacs)
 	// the following pointer uninitalized (preventing the DB from being deleted at the
 	// end of processing this command.
 	DB * db = nullptr;
-	
-	int device_index = 0;
-	int buffer_offset = 0;
-
-	if (buffer[0] >= '0' && buffer[0] <= '9')
-	{
-		device_index = buffer[0] - '0';
-		buffer_offset = 2;
-	}
 
+/*
+		LOG(_log_, nullptr);
+					google::protobuf::Map<string, string> * result = r.mutable_results();
+		LOG(_log_, nullptr);
+		// LEFT OFF HERE
+					(*result)[string("happy")] = string("joy");
+		LOG(_log_, nullptr);
+					string s;
+					bool outcome = r.SerializeToString(&s);
+					if (!outcome)
+						throw string("WHO_DEVICE bad serialize");
+		LOG(_log_, nullptr);
+*/
+	
+	
 	try
 	{
-		if (device_index < 0 || device_index >= ndacs)
+		GenericPB g;
+		//LOG(_log_, nullptr);
```

Comment:
I've thought of that but there's not "good" way of getting fine granularity control. Let me think about this - do you have thoughts?